src/test/resources/com/sap/oss/phosphor/fosstars/model/score/oss/DependabotScoreTestVectors.yml

```diff
@@ -104,10 +104,10 @@ elements:
         - "JAVA"
   expectedScore:
     type: "DoubleInterval"
-    from: 6.0
+    from: 1.0
     openLeft: false
     negativeInfinity: false
-    to: 9.0
+    to: 3.0
```

Comment:
This cannot be changed like this, As the Okay rating comes only between range 5 - 9